### ISSUE TITLE
Strategic JSON patch no longer supported?

### ISSUE THREAD

**Issue by brendanhay** [kubernetes-client/haskell#83]
Comparing `0.3.0.0` and `3.1.0.0`, the following instances have been _removed_ from the newer version:

```
FromJSON a => MimeRender MimeStrategicMergePatchjson a
FromJSON a => MimeRender MimeMergePatchjson a
FromJSON a => MimeRender MimeJsonPatchjson a
...
```

While I'm unaware of the reasoning behind the removal, it does appear that calls such as `patchNamespacedService` and all other PATCH operations are borderline useless, as you're now required to assemble a read+patch to formulate an incremental atomic update. The only remaining instance that satisfies the PATCH operation's `MimeRender contentType Body` constraint is:

```
instance ToJSON a => MimeRender MimeJSON a
```

TL;DR: It appears that strategic patches are now unsupported in `>= 3.1.0.0`.

**Comment by jonschoning** [comment]
this has been fixed
